docs-internal/features/package-info.java

```diff
@@ -0,0 +1,18 @@
+@Feature(value = "Security/Providers/ABAC/Validators/EL",
+        description = "Expression language validator for ABAC security provider.");
+@HelidonFlavor("SE", "MP")
+package io.helidon.security.abac.el {
+
+}
+
+@Features(
+    @Feature(value = "Security/Providers/Digest-Auth",
```

Comment:
The "path" is declared here and in the catalog as a YAML tree.
This feels like it's needed for the startup list since it would be an over-head to read the catalog just to render the tree structure.

I'd like to see if we can get rid of the path and instead have each feature be defined with a unique id and the feature tree be defined only in the catalog.

This would require the startup list to read the catalog to render the feature tree.

If the startup list is optional and a feature itself (i.e a separate modules, a drop-in JAR for MP, and a mini API for the main class in SE), then the feature-startup-list module would depend on the catalog and would have access to it (perhaps it bundles it).

However, reading the catalog would add a bit of overhead, especially if it's JSON. I guess we can optimize this usecase, maybe with code generation based on the YAML, or by generating a file with an optimized format.